app/lib/actions/katello/upstream_subscriptions/bind_entitlements.rb

```diff
@@ -0,0 +1,28 @@
+module Actions
+  module Katello
+    module UpstreamSubscriptions
+      class BindEntitlements < Actions::Base
+        middleware.use Actions::Middleware::KeepCurrentTaxonomies
+
+        def plan(pools = [])
+          fail _("No pools were provided.") unless pools.any?
```

Comment:
Do you mind adding a test for these two errors we might raise? You can basically copy from my remove entitlements action test :)